ui/perfherder/perf-helpers/helpers.js

```diff
@@ -888,3 +889,60 @@ export const getResultsMap = (results) => {
   });
   return { testNames, names, platforms, resultsMap };
 };
+
+export const getTimeDifference = (currentDate, dueDate) => {
+  const timeDifference = Math.abs(dueDate - currentDate);
+  let differenceInDays = Math.ceil(timeDifference / (1000 * 60 * 60 * 24));
+  // saturday and sunday are considered weekend days
+  const weekendDaysCount = 2;
+  // convert days to weeks and keeps the count of full weeks
+  const weeksCount = Math.trunc(differenceInDays / 7);
+  // count the total weekend days
+  const weekendDaysToSubstract = weekendDaysCount * weeksCount;
+
+  // [1. Mon] [2. Tue] [3. Wed] [4. Thu]  [5. Fri] [6. Sat] [7. Sun]
+  //                            startDate -------- -------- ----->>>
+  // -------- -------> endDate
```

Comment:
```suggestion
  //                            currentDate -------- -------- ----->>>
  // -------- -------> dueDate
```
So that it corresponds with the rest of the logic.